clouddriver-cloudrun/src/test/java/com/netflix/spinnaker/clouddriver/cloudrun/deploy/converters/CloudrunDeployManifestDescriptionTest.java

```diff
@@ -0,0 +1,43 @@
+package com.netflix.spinnaker.clouddriver.cloudrun.deploy.converters;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.*;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.netflix.spinnaker.clouddriver.cloudrun.converter.manifest.CloudrunDeployManifestConverter;
+import com.netflix.spinnaker.clouddriver.cloudrun.op.manifest.CloudrunDeployManifestOperation;
+import com.netflix.spinnaker.clouddriver.cloudrun.security.CloudrunNamedAccountCredentials;
+import com.netflix.spinnaker.credentials.CredentialsRepository;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Before;
+import org.junit.Test;
+
+public class CloudrunDeployManifestDescriptionTest {
+
+  CloudrunDeployManifestConverter converter;
+  CredentialsRepository<CloudrunNamedAccountCredentials> credentialsRepository;
+  CloudrunNamedAccountCredentials mockCredentials;
+  Map<String, Object> input =
+      new HashMap<>() {
+        {
+          put("accountName", "cloudrunaccount");
+        }
+      };
+
+  @Before
+  public void init() {
+    converter = new CloudrunDeployManifestConverter();
+    credentialsRepository = mock(CredentialsRepository.class);
+    converter.setCredentialsRepository(credentialsRepository);
+    converter.setObjectMapper(new ObjectMapper());
+    mockCredentials = mock(CloudrunNamedAccountCredentials.class);
+  }
+
+  @Test
+  public void ConvertOperationTest() {
+
+    when(credentialsRepository.getOne(any())).thenReturn(mockCredentials);
+    assertTrue(converter.convertOperation(input) instanceof CloudrunDeployManifestOperation);
```

Comment:
This test doesn't feel very comprehensive.

Reply:
@mattgogerly I will look into it. Might take some time.

Reply:
@mattgogerly ..can't do much here. Still added one more test case.

Reply:
@mattgogerly ...
Please can you review the changes

Reply:
@mattgogerly Please can you review the changes?

Reply:
@mattgogerly  Please can you review the changes?

Reply:
@mattgogerly Please can you review the changes?